src/install/db.py

```diff
@@ -4,14 +4,15 @@
 from shlex import split
 from subprocess import PIPE, CalledProcessError, run
 
-from helperFunctions.install import InstallationError, OperateInDirectory
+from helperFunctions.install import InstallationError, OperateInDirectory, check_distribution
 
 
-def install_postgres(version: int = 14):
+def install_postgres(version: int = 16):
```

Comment:
This has to also be used for the `postgres_is_installaed` function, since no ugrade to 16 will be done on existing 14 installation.

Reply:
I added a version check with a warning it the installed version is outdated